pkg/controller/datavolume-controller.go

```diff
@@ -722,10 +725,15 @@ func (r *DatavolumeReconciler) ensureExtendedToken(pvc *corev1.PersistentVolumeC
 }
 
 func (r *DatavolumeReconciler) selectCloneStrategy(datavolume *cdiv1.DataVolume, pvcSpec *corev1.PersistentVolumeClaimSpec) (cloneStrategy, error) {
-	if datavolume.Spec.Source.PVC == nil {
+	isClone := datavolume.Spec.Source.PVC != nil
```

Comment:
Not sure this var adds readability, so either remove it or at least reduce it scope:
`if isClone := datavolume.Spec.Source.PVC != nil; !isClone {...}`

Reply:
In my opinion it really helps, it is not clear from the first look that `PVC != Nil` means it is clone. Of course if you stop and think about it it makes sense. But having it explicitly named makes code easier to quickly read.